docs/InstallGuide.md

```diff
@@ -131,294 +158,206 @@ To automatically launch the mysql server, execute the following command.
-+ Kernel 2.6.32-358.6.2.el6.x86_64
+```
+# ./bin/vnctl datapaths add --uuid dp-test1 --display-name test1 --dpid 0x0000aaaaaaaaaaaa --node-id vna
+```
 
+* dpid
 
-Installation and Setup of OpenVNet
------------------------------------
+The datapath ID specified in `/etc/sysconfig/network-scripts/ifcfg-br0`
 
-### Network interfaces setup
+* node-id
 
-We're going to enslave eth0 to openvswitch so OpenVNet can use it. You will no longer be able to use eth0 to ssh into the node or access the Internet. Prepare another interface for that if you need to.
+The ID of the vna written in `/etc/openvnet/vna.conf`
 
-    # vi /etc/sysconfig/network-scripts/ifcfg-eth0
-    DEVICE=eth0
-    DEVICETYPE=ovs
-    TYPE=OVSPort
-    OVS_BRIDGE=br0
-    BOOTPROTO=none
-    ONBOOT=yes
-    HOTPLUG=no
-    #
-    # vi /etc/sysconfig/network-scripts/ifcfg-br0
-    DEVICE=br0
-    DEVICETYPE=ovs
-    TYPE=OVSBridge
-    ONBOOT=yes
-    BOOTPROTO=static
-    IPADDR=172.16.20.41
-    NETMASK=255.255.255.0
-    HOTPLUG=no
-    OVS_EXTRA="
-     set bridge     ${DEVICE} protocols=OpenFlow10,OpenFlow12,OpenFlow13 --
-     set bridge     ${DEVICE} other_config:disable-in-band=true --
-     set bridge     ${DEVICE} other-config:datapath-id=00004e6d2b508f4c --
-     set bridge     ${DEVICE} other-config:hwaddr=02:01:00:00:00:01 --
-     set-fail-mode  ${DEVICE} standalone --
-     set-controller ${DEVICE} tcp:127.0.0.1:6633
-    "
 
-### OpenVNet package installation
+#### Network
 
-Please install OpenVNet according to the "Pre-setup" and "OpenVNet installation" sections of "Preliminary Operations and Installation".
+```
+# ./vnctl networks add --uuid nw-test1 --display-name testnet1 --ipv4-network 10.100.0.0 --ipv4-prefix 24 --network-mode virtual
+```
 
-The config files (vnmgr.conf, vna.conf and webapi.conf) in the /etc/openvnet directory do not need to be edited. They're set up for a single node installation by default.
+* ipv4-network
 
-### Other package installation
+The IPv4 network address.
 
-    # yum install qemu-kvm
-    # yum install tcpdump
+* ipv4-prefix
 
+The IPv4 network prefix. (default 24)
 
-Start the required processes for OpenVNet
------------------------------------------
+* network-mode
 
-### redis
+The mode of the network to create. Use `virtual` in case of creating virtual network.
 
-    # chkconfig --list redis
-    redis           0:off   1:off   2:off   3:off   4:off   5:off   6:off
-    # chkconfig redis on
-    # chkconfig --list redis
-    redis           0:off   1:off   2:on    3:on    4:on    5:on    6:off
-    # service redis start
+#### Interface
 
+```
+# ./vnctl interfaces add --uuid if-inst1 --mode vif --owner-datapath-uuid dp-test1 --mac-address 10:54:ff:00:00:01 --network-uuid nw-vnet1 --ipv4-address 10.100.0.10 --port-name inst1
+# ./vnctl interfaces add --uuid if-inst2 --mode vif --owner-datapath-uuid dp-test1 --mac-address 10:54:ff:00:00:02 --network-uuid nw-vnet1 --ipv4-address 10.100.0.11 --port-name inst2
+```
```

Comment:
We just created `nw-test1` but now we are using `nw-vnet1`. This causes an UnknownUUIDResource error.